ttyplot.1

```diff
@@ -53,6 +53,14 @@ as a hard value limit after which an error line will be drawn
 see
 .Fl e
 .Pc .
+.It Fl M Ar hardmin
+Use
+.Ar hardmin
+as a hard value limit before which an error line will be drawn
```

Comment:
this is also rather unfortunate explanation, please word it better and explain what exactly will go on

Reply:
Improved manual a bit. Inform me if any more changes are needed.